.github/workflows/deploy_ecs_relayers.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy ECS relayers
```

Comment:
Please consider using the file [deploy_testnet.yml](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_testnet.yml), which is the modified file I've merged in the mentioned PR above.

In that file you'll find some other changes I've implemented, such as: `ECR_REPOSITORY` is now:  `relayer`, the docker tag to be pushed (TAG) is now: `stable`, and also the [use of matrix](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_testnet.yml#L54) to deploy all the relayers in the same workflow.